src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAO.scala

```diff
@@ -123,6 +130,47 @@ class ElasticSearchDAO(client: TransportClient, indexName: String, ontologyDAO:
     populateSuggestions(client, indexName, field, text)
   }
 
+  override def statistics: LogitMetric = {
+    if (FireCloudConfig.Metrics.libraryNamespaces.isEmpty)
+      throw new FireCloudException("no namespaces defined for ElasticSearchDAO.statistics()")
+
+    val AGGKEY = "sumSamples"
+
+    // build query: any namespace defined in our conf file
+    val namespaceFilter = boolQuery()
+    FireCloudConfig.Metrics.libraryNamespaces.map { ns =>
+      namespaceFilter.should(termQuery("namespace.keyword", ns))
```

Comment:
I think we want `prefixQuery()` instead, so we don't match e.g. `joel-broad-genomics-data`

Reply:
hmm, I disagree. [`termQuery`](https://www.elastic.co/guide/en/elasticsearch/client/java-api/5.4/java-term-level-queries.html#java-query-dsl-term-query) is an exact-match against the entire value, without any spell correction or stemming or prefixing/wildcarding - I think that's what we want here.

Reply:
If it's exact-match then I agree with you.  I was interpreting the `TermQueryBuilder` javadoc "A Query that matches documents containing a term" as a contains-match.

Reply:
yeah, technically ... the document contains many fields; one of those fields must be an exact-match on the term criteria. So technically the document contains the term ... but we have the ability to do exact-match, and specify which field we care about.